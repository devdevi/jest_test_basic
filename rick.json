{
    "id": 1,
    "name": "Rick Sanchez",
    "status": "ALive",
    "species": "Humano",
    "gender": "Masculino"
}