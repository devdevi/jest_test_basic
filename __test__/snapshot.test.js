import { getCharacter } from '../snapshot';
import rick from '../rick.json';

describe('Es hora de las instantaneas', () => {
    test('Snapshot', () => {
        expect(getCharacter(rick)).toMatchSnapshot();
    });

    test('Fallar el snapshot', () => {
        const user = {
            id: Math.floor(Math.random() * 20),
            name: 'Alejandro Gonz√°lez Reyes',
            profesion: 'JavaScript and PHP Development',
            createdAt: new Date()
          }
    expect(user).toMatchSnapshot({
            id: expect.any(Number),
            createdAt: expect.any(Date)
        });
    })
})