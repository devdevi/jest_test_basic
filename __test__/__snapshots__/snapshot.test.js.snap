// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Es hora de las instantaneas Fallar el snapshot 1`] = `
Object {
  "createdAt": Any<Date>,
  "id": Any<Number>,
  "name": "Alejandro Gonz√°lez Reyes",
  "profesion": "JavaScript and PHP Development",
}
`;

exports[`Es hora de las instantaneas Snapshot 1`] = `
Object {
  "gender": "Masculino",
  "id": 1,
  "name": "Rick Sanchez",
  "species": "Humano",
  "status": "ALive",
}
`;
